StringCalc/CalculatorHelpers.cs

```diff
@@ -0,0 +1,65 @@
+﻿namespace StringCalc
+{
+    public static class CalculatorHelpers
+    {
+        private static readonly string[] Seperators = { "\n", ","};
+        private static readonly string[] DelimiterStrings = { "//", "[", "][", "]" };
+        public enum Defaults
```

Comment:
This is an interesting use of enums... What was your thinking here over using separate constant variables for each value?